contracts/andromeda_gumball/Cargo.toml

```diff
@@ -0,0 +1,34 @@
+[package]
+name = "andromeda_gumball"
+version = "0.1.0"
+edition = "2018"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+
+[lib]
+crate-type = ["cdylib", "rlib"]
+
+[features]
+# for more explicit tests, cargo test --features=backtraces
+backtraces = ["cosmwasm-std/backtraces"]
+# use library feature to disable all instantiate/execute/query exports
+library = []
+
+[dependencies]
+cosmwasm-std = { version = "0.16.2" }
+cw-storage-plus = "0.9.1"
+schemars = "0.8.3"
+serde = { version = "1.0.127", default-features = false, features = ["derive"] }
+thiserror = { version = "1.0.26" }
+cw0 = "0.9.1"
+cw721 = "0.9.1"
+cw2 = "0.9.1"
+base64 = "0.13.0"
+terrand = { version = "3.2.2", features = ["library"] }
+andromeda-protocol = { version = "0.1.0", path = "../../packages/andromeda_protocol" }
+common = { version = "0.1.0", path = "../../packages/common" }
+ado-base = { path = "../../packages/ado_base", version = "0.1.0", features = ["modules"] }
```

Comment:
nit: It doesn't look like we are using modules here so we should remove the `features` attribute.